FirebaseAppDistribution/Sources/FIRAppDistributionUIService.m

```diff
@@ -230,14 +228,17 @@ - (void)startFeedbackWithAdditionalFormText:(NSString *)additionalFormText
                                                                 // UIService to Swift.
                                                                 [self resetUIState];
                                                               }];
-  [self initializeUIState];
-  [self.hostingViewController presentViewController:feedbackViewController
-                                           animated:YES
-                                         completion:nil];
+
+  // TODO: Potentially move the check for the release version here.
+  if (feedbackViewController != nil) {
```

Comment:
I think at this point we don't want to show the feedback view controller if the release hasn't been identified. I _think_ this would mirror the behavior in Android.